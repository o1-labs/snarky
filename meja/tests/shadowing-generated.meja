type t = field;

type var = field;
