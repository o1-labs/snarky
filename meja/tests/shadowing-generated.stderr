File "tests/shadowing-generated.meja", line 3, characters 5-8:
Error: There is already a generated type with name var.
